apps/dashboard/src/@3rdweb-sdk/react/hooks/useEngine.ts

```diff
@@ -273,12 +274,13 @@ export function useEngineDeleteCloudHosted() {
 
   return useMutation({
     mutationFn: async ({
+      engineId,
       deploymentId,
       reason,
       feedback,
     }: DeleteCloudHostedInput) => {
       const res = await fetch(
-        `${THIRDWEB_API_HOST}/v2/engine/deployments/${deploymentId}/infrastructure/delete`,
+        `${THIRDWEB_API_HOST}/v2/engine/deployments/${deploymentId}/infrastructure/delete?engineId=${engineId}`,
```

Comment:
feels a little weird to have this in a query param for a POST request?

also isn't a deploymentId linked to an engineId already ?

Reply:
Let me handle this from the API server end. It's less explicit but should it should resolve for for most users and is the correct longer term approach.